document-generator-company-report/src/test/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/ApiToPreviousNamesMapperTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.companieshouse.document.generator.company.report.mapping.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.model.company.PreviousCompanyNamesApi;
+import uk.gov.companieshouse.document.generator.company.report.mapping.mappers.previousnames.ApiToPreviousNamesMapper;
+import uk.gov.companieshouse.document.generator.company.report.mapping.mappers.previousnames.ApiToPreviousNamesMapperImpl;
+import uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.previousnames.PreviousNames;
+
+@ExtendWith({MockitoExtension.class})
+public class ApiToPreviousNamesMapperTest {
```

Comment:
We usually use @TestInstance(Lifecycle.PER_CLASS) so a new test instance will be created once per test class. Rather then multiple.